neuron/src/lib/Neuron/Web/Query/View.hs

```diff
@@ -121,6 +121,7 @@ renderZettelLink conn (fromMaybe def -> linkView) Zettel {..} = do
             else Just $ "Tags: " <> T.intercalate "; " (unTag <$> zettelTags)
     elAttr "span" ("class" =: "zettel-link" <> withTooltip linkTooltip) $ do
       neuronRouteLink (Some $ Route_Zettel zettelID) mempty $ text zettelTitle
+      el "sup" $ text "ᛦ"
```

Comment:
Can this be done in CSS `content`? I imagine you have an empty `sup` element, and style the content in CSS ... 

Otherwise you would have to use `noSnippet` from #326 in <sup> to suppress it from search preview.

Reply:
I experimented with doing it in CSS. I found it unsatisfactory trying to get the sizing and positioning correct in a way that behaved well if the browser screen was resized. Doing it in the CSS also prevents it from being highlighted if someone copies the text (can be a pro or a con depending on your view. I personally liked it being physically in the text)

Really, though, the main consideration for having it in the HTML was that it was easier to implement and is more easy to make accessible to screen readers. Had to get the noSnippet by placing the text inside an inner span because google only supports noSnippet on `div`, `section`, and `span`.